install.sh

```diff
@@ -281,8 +282,9 @@ add_tags_to_asg_resources
 # Resume bamboo server if the credential is provided
 resume_bamboo_server
 
+# Print information about manually adding the new k8s context
+set_current_context_k8s
```

Comment:
🤦 sorry I made this change locally and thought that I have merged it. Thanks for fixing it!